Smartling.API/Job/JobApiClient.cs

```diff
@@ -78,35 +78,37 @@ public virtual AddFileToJobResponse AddLocale(string localeId, string jobId, boo
       return JsonConvert.DeserializeObject<AddFileToJobResponse>(response["response"]["data"].ToString());
     }
 
-    public virtual List<Smartling.Api.Model.Job> Get(string jobName = "")
+    public virtual List<Smartling.Api.Model.Job> Get(string jobName = "", string statusFilter = "")
     {
-      var page = GetPage(jobName, PageSize, 0);
+      var page = GetPage(jobName, PageSize, 0, statusFilter);
       var results = new List<Smartling.Api.Model.Job>();
       var pageNumber = 0;
       results.AddRange(page.items);
 
       while (page.totalCount > results.Count && page.items.Count > 0)
       {
         pageNumber++;
-        page = GetPage(jobName, PageSize, PageSize * pageNumber);
+        page = GetPage(jobName, PageSize, PageSize * pageNumber, statusFilter);
         results.AddRange(page.items);
       }
 
       return results;
     }
-
-    public virtual JobList GetPage(string jobName, int limit, int offset)
+    
+    public virtual JobList GetPage(string jobName, int limit, int offset, string statusFilter = "")
     {
-      StringBuilder uriBuilder;
-      if (string.IsNullOrEmpty(jobName))
+      var url = string.Format(GetJobUrl, projectId, limit, offset);
+      if (!string.IsNullOrEmpty(jobName))
       {
-        uriBuilder = this.GetRequestStringBuilder(string.Format(GetJobUrl, projectId, limit, offset));
+        url += "&jobName=" + jobName;
```

Comment:
Don't forget to urlencode all params (`jobName`)

Reply:
Done